docs/sources/administration/planning-iam-strategy/index.md

```diff
@@ -0,0 +1,213 @@
+
+Grafana allows for precise permission settings on both dashboards and folders, giving you the ability to control which users and teams can view, edit, and administer them.
+For example, you might want a certain viewer to be able to edit a dashboard. While that user can see all dashboards, you can grant them access to update only one of them.
+
+In [Grafana Enterprise]({{< relref "../../introduction/grafana-enterprise" >}}), you can also grant granular permissions for data sources to control who can query and edit them.
+
+Dashboard, folder, and data source permissions can be set through the UI or APIs or provisioned through Terraform.
+
+### Role-based access control
+
+{{% admonition type="note" %}}
+Available in [Grafana Enterprise]({{< relref "../../../introduction/grafana-enterprise" >}}) and [Grafana Cloud](/docs/grafana-cloud/).
+{{% /admonition %}}
+
+If you feel that the basic organization and server administrator roles are too limiting, it may be beneficial to employ RBAC.
+RBAC is a flexible approach to managing user access to Grafana resources, including users, data sources, and reports. It enables easy granting, changing, and revoking of read and write access for users.
+
+RBAC comes with pre-defined roles, such as data source writer, which allows updating, reading, or querying all data sources.
+You can assign these roles to users, teams, and service accounts.
+
+In addition, RBAC empowers you to generate personalized roles and modify permissions authorized by the standard Grafana roles.
+
+## User synchronization between Grafana and Identity Providers
+
+When connecting Grafana to an identity provider, it's important to think beyond just the initial authentication setup. You should also think about the maintenance of user bases and roles. Using Grafana's team and role synchronization features ensures that updates you make to a user in your identity provider will be reflected in their role assignment and team memberships in Grafana.
+
+### Team sync
+
+Team sync is a feature that allows you to synchronize teams or groups from your authentication provider with teams in Grafana. This means that users of specific teams or groups in LDAP, OAuth, or SAML will be automatically added or removed as members of corresponding teams in Grafana. Whenever a user logs in, Grafana will check for any changes in the teams or groups of the authentication provider and update the user's teams in Grafana accordingly. This makes it easy to manage user permissions across multiple systems.
+
+{{% admonition type="note" %}}
+Available in [Grafana Enterprise]({{< relref "../../introduction/grafana-enterprise" >}}) and [Grafana Cloud Advanced](/docs/grafana-cloud/).
+{{% /admonition %}}
+
+{{% admonition type="note" %}}
+Team synchronization occurs only when a user logs in. However, if you are using LDAP, it is possible to enable active background synchronization. This allows for the continuous synchronization of teams.
+{{% /admonition %}}
+
+### Role Sync
```

Comment:
@linoman Are you planning to add content here?

Reply:
Yes! I forgot this section was missing information.